### PR TITLE
(internal) release_patch: build v5.6.2535

### PR DESCRIPTION
**Pull request by Chickensoupwithrice**
Test plan: automated release PR, CI will perform additional checks

### CODE REVIEW

**Comment by sourcegraph-buildkite** [comment]
- :green_circle: Internal release is ready for promotion!
- :warning: Do not close/merge the pull request or delete the associated branch if you intend to promote it.

**Comment by sourcegraph-buildkite** [comment]
- :green_circle: Release has been promoted, see tag: https://github.com/sourcegraph/deploy-sourcegraph-helm/tree/v5.6.2535.
- :no_entry: Do not under any circumstance delete the branch holding the tagged commit (i.e. `release-v5.6.2535`).
- :arrow_right: You can safely close the PR and delete its a associated branch.

File: charts/sourcegraph-appliance/values.yaml [hunk 1]
```diff
@@ -57,8 +57,7 @@ ingress:
   enabled: false
   className: ""
   annotations:
-    {}
-    # kubernetes.io/ingress.class: nginx
+    kubernetes.io/ingress.class: alb
```
Comment (craigfurman) [inline charts/sourcegraph-appliance/values.yaml:60]: y tho

**Review by craigfurman — COMMENTED** [review]
@sourcegraph/release just noticed that sadly, the release.yaml scripting added in https://github.com/sourcegraph/deploy-sourcegraph-helm/pull/534 did not cause a new appliance Helm chart version to be released. I'm puzzled as to why this is 🤔 

https://github.com/sourcegraph/deploy-sourcegraph-helm/actions/runs/10499469029/job/29086296159 - we can see that appliance 0.1.0 is being re-released, presumably because the comby expressions that change the chart version were not running as we expected?

**Comment by craigfurman** [comment]
Ah, mystery solved: yesterday's release branched off an older commit in this repo, not the latest main. Next time, if we the helm chart from latest main, the release.yaml changes that enable appliance chart releases will execute and hopefully release a well-formed chart.

File: charts/sourcegraph-appliance/values.yaml (hunk 1 above)
Comment (craigfurman) [reply charts/sourcegraph-appliance/values.yaml:60]: this is still a mystery AFAIK

Comment (Chickensoupwithrice) [reply charts/sourcegraph-appliance/values.yaml:60]: I think I forgot to change branch / didn't clean up my changes

**Comment by Chickensoupwithrice** [comment]
I'll re-do the release today, looks like I need to clean up some of the changes and run from main

**Pull request closed without merging**